awx/ui_next/src/util/prompt/getCredentialPasswords.js

```diff
@@ -0,0 +1,29 @@
+export default function getCredentialPasswords(values) {
```

Comment:
This could possibly be shortened or maybe even eliminated if we can avoid created keys in the formik values object that the api doesn't recognize.